Basic Rules.txt

```diff
@@ -597,18 +597,27 @@ you below the minimum miss chance.
 A character occupies 1m of space. however, as Compound X is not played out on a 
 squares-board, usually it doesn't matter if two players are within the same 
 1m space.
-	Two Players locked in melee combat are considered to be in adjacent 
-1m spaces, not in the same 1m space. Have you ever seen a fight? you don't start 
-face-to-face unless you're at a bar.
+	Two Players locked in melee combat are considered to be in adjacent 1m spaces, 
+not in the same 1m space. Have you ever seen a fight? you don't start face-to-face 
+unless you're at a bar.
 	Generally speaking, once a character has initiated melee combat with another 
-character, they are both considered to be in "melee combat," and occupy adjacent 
-1m spaces. At this point (unless one character has a CQB weapon) both characters 
-are assumed to be using their fists or other melee weapons. If a character wishes 
-to leave melee combat, they incur a "penalty attack."  Characters also incur 
-"penalty attacks" when moving through or next to an enemy character's 1m space,
-or within range of a melee weapon that the enemy has equipped and when performing 
-a skill check when in melee combat.
-	Melee combat is relatively simple. To hit an enemy player: add your dexterity 
+opponent, they are both considered to be in "melee combat," and occupy spaces within
+reach of one of the combatant's melee weapons. At this point (unless one character 
+has a CQB weapon) both characters are assumed to be using their fists or other melee 
+weapons. If a character wishes to leave melee combat, they incur an "Opportunity Attack."  
+	
+Opportunity Attack:
+A free attack* that is triggered when an opponent leaves melee combat or walks adjacent 
+(within melee range) to a character that is aware of their opponent. Characters 
+also incur Opportunity Attacks by performing a non-combat skill check when locked in melee 
+combat (such as trying to repair their own armor) or by trying to shoot a different opponent
+with a non-CQB weapon. 
+	*The free attack can be made with a CQB weapon or a melee weapon (depending on if the weapon
+	is already equipped in the character's hands). If the free attack is made with a CQB weapon,
+	and the weapon is not already aimed at the target, standard Hip Firing rules apply.

```

Comment:
I really like all of this. This is way more clear than it was before and I think someone who hadn't played DnD could understand how Opportunity Attacks work. Great Job!